libqtile/widget/chord.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2014 Sean Vig
+# Copyright (c) 2014 roger
+# Copyright (c) 2014 Adi Sieker
+# Copyright (c) 2014 Tycho Andersen
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+from libqtile import bar, hook
+from libqtile.widget import base
+
+
+class Chord(base._TextBox):
+    """Display current key chord"""
+    orientations = base.ORIENTATION_HORIZONTAL
+    defaults = [
+        ("chords_colors", {},
```

Comment:
Is there a reason to have this config value? People should be able to set foreground and background themselves right?

Reply:
Ya, it's inspired by Vim statusbars - in theme we can change colors of mode(in here chord) indicators eg. Insert is orange and normal is blue. This value let exactly do that.

Reply:
But I don't think this code is necessary; can't people set `background=...` when configuring their widget?

Reply:
Yes, they can but case here is to have ability to assign different colors to different modes, on your suggestion(and current behavior) colors are static and applied to all modes. This add instant visual clue about mode without even need to read text

Reply:
Oh, I see! Yes, thank you.